src/components/shared/components/Icons/components/IconCoffeeCup.tsx

```
@@ -0,0 +1,25 @@
+import clsx from "clsx";
+import { SVGProps } from "react";
+
+const IconCoffeeCup = <T extends unknown>({
```

Biome (1.9.4)

[error] Constraining a type parameter to any or unknown is useless.

All types are subtypes of any and unknown.
Safe fix: Remove the constraint.


(lint/complexity/noUselessTypeConstraint)